packages/components/MemberList/components/RevokeAccessControl.js

```diff
@@ -15,16 +17,32 @@ const RevokeButton = styled.button`
 `;
 
 const MinusIcon = styled.div`
-  width: 14px;
+  width: 12px;
   height: 2px;
   background-color: ${({ theme }) => theme.color.white};
   cursor: pointer;
 `;
 
-const RevokeAccessControlComponent = ({ onClickRevoke, className }) => (
-  <RevokeButton className={className} onClick={onClickRevoke}>
-    <MinusIcon />
-  </RevokeButton>
-);
+const ResendIcon = styled(Icon)`
+  width: 20px;
+  height: 20px;
+`;
+
+const RevokeAccessControlComponent = ({
+  isAddedToRevoke,
+  onClickRevoke,
+  className,
+}) => isAddedToRevoke
+  ? (
+    <Hint text="Resend" position="center_left">
```

Comment:
pls, use `import { HINT_POSITION } from '@caesar/components';` instead of 'center_left'

Reply:
Done